src/lib/components/NotificationList.svelte

```diff
@@ -48,13 +48,48 @@
 </script>
 
 <ul
-  class={`fixed p-6 space-y-4 overflow-y-auto max-h-[calc(100vh-54px)] ${className} ${getPositionClasses(
-    position
-  )}`}
-  class:z-10={notificationsList?.length}
+  class={` ${className} ${getPositionClasses(position)}`}
+  class:z-index={notificationsList?.length}
```

Comment:
Given that we are using CSS we can now use the [`class:name` element directive](https://svelte.dev/docs/element-directives#class-name):

```suggestion
  class={className}
  class:position-top-right={possition === Position.TopRight}
  class:z-index={notificationsList?.length}
```

Reply:
Yeah sure!